.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Oops probably don't want to add your vscode settings here

Reply:
Oops. Fixed